File: app/src/main/java/com/caramelheaven/lennach/ui/main/MainActivity.java
package com.caramelheaven.lennach.ui.main;

import android.inputmethodservice.KeyboardView;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.view.KeyEvent;

import com.caramelheaven.lennach.R;
import com.caramelheaven.lennach.ui.board.BoardFragment;

import timber.log.Timber;

public class MainActivity extends AppCompatActivity {

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        if (savedInstanceState == null) {
            getSupportFragmentManager()
                    .beginTransaction()
                    .replace(R.id.fragment_container, BoardFragment.newInstance("b"))
                    .commit();
        }
    }
}

File: app/src/main/java/com/caramelheaven/lennach/ui/main/MainActivity.kt
package com.caramelheaven.lennach.ui.main

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.caramelheaven.lennach.R
import com.caramelheaven.lennach.ui.board.BoardFragment

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val temp = intent.getStringExtra("ITEM");

        if (savedInstanceState == null) {
            supportFragmentManager
                    .beginTransaction()
                    .replace(R.id.fragment_container, BoardFragment.newInstance(temp))
                    .commit()
        }
    }
}
